docs/Additional Information/hackathon-rules-waiver-release-and-code-of-conduct.md

```diff
@@ -1,98 +0,0 @@
----
-layout: nodes.liquid
```

Comment:
Don't we need this for our future hackathons and events? @PatrickAlphaC @pappas999